macrosynergy/pnl/historic_portfolio_volatility.py

```diff
@@ -125,23 +125,32 @@ def _mask_nans(
     remove_zeros: bool,
 ) -> pd.DataFrame:
     """Mask NaNs in the dataframe"""
+    lback_periods = 0 if lback_periods < 0 else lback_periods
```

Comment:
Does this work with our `lback_periods = -1` for full sample or is the logic that 0 or less is full sample?

Reply:
No, the only acceptable negative value is -1.

https://github.com/macrosynergy/macrosynergy/blob/d604e10c97a61d33744e86ba6611ef8e783a0482/macrosynergy/pnl/historic_portfolio_volatility.py#L205